en/orm/validation.rst

```diff
@@ -377,6 +377,27 @@ can help provide a nicer user experience. Because of this CakePHP includes an
 The fields to check existence against in the related table must be part of the
 primary key.
 
+Association Count Rules
+-----------------------
+
+If you need to validate that a property or association contains the correct
+number of values, you can use the ``validCount()`` rule::
+
+    // No more than 5 tags on an article.
+    $rules->add($rules->validCount('tags', 5, '<=', 'You can only have 5 tags'));
+
+When defining count based rules, the 3 parameter lets you define the comparison
```

Comment:
3 = three or third?
Better write it out.

If it's three, then it should be parameter**S**.